src/sentry/snuba/discover.py

```diff
@@ -960,6 +960,7 @@ def measurements_histogram_query(
     precision=0,
     min_value=None,
     max_value=None,
+    data_filter="upper_outer_fence",
```

Comment:
Can this default to `None`? The default value for the RUM page should be on the front end.

Also, for consistency sake, we should consider using`p95` as the default.